webui/src/components/RepositoryCreateForm.js

```diff
@@ -52,7 +54,7 @@ export const RepositoryCreateForm = connect(({ repositories }) => {
             <Form.Group as={Row}>
                 <Form.Label column sm={fieldNameOffset}>Storage Namespace</Form.Label>
                     <Col sm={sm}>
-                        <Form.Control type="text" ref={storageNamespaceField} placeholder="i.e. s3://example-bucket/" onChange={checkValidity}/>
+                        <Form.Control type="text" ref={storageNamespaceField} placeholder={`i.e. ${config.payload['blockstore.type']}://example-bucket/`} onChange={checkValidity}/>
```

Comment:
```suggestion
                        <Form.Control type="text" ref={storageNamespaceField} placeholder={`${config.payload['blockstore.type']}://example-bucket/`} onChange={checkValidity}/>
```
I know we have *too many of these PRs* for switching "e.g." and "i.e." (@johnnyaug), but let's fix this one while we're here.